software/cloud-dashboard/node-server/config/device-map.js

```diff
@@ -6,7 +6,7 @@
 module.exports={
 	locMap : {
 	"4b0031000d51353432393339": "ADPLKenyaC6304",
-        "300057000951343334363138": "ADPLKenyaN3763",
+        "3e001d000e51353432393339": "ADPLKenyaN6833",
```

Comment:
Yeah, that sounds good! I can run a backfill on the database to update all the old records at some point too hopefully this week.